package.json

```diff
@@ -9,7 +9,7 @@
     "test": "karma start test/config/karma.unit.js",
     "distro": "node tasks/build-distro.js",
     "test:distro": "node tasks/test-distro.js",
-    "preversion": "run-s distro test:distro"
+    "prepublishOnly": "run-s distro test:distro"
```

Comment:
The reason for the inverse change (https://github.com/bpmn-io/bpmn-js/commit/bd0e86d3136d14082636f6b1af7c759471d6351c) was that publishing failed with `prepublishOnly`: The OTP publish token would always timeout before the bundle completed. 

Maybe we can hook into another step _or_ simplify the build step to not include the additional tests.

Reply:
Thanks for the insight. We could then use `postversion` instead. It will still be run on publish but before `npm publish` step.

Reply:
Works for me, if it works for `np` + OTP, too. :+1:

Reply:
It should be as `npm publish` is a separate step and only then one is asked for OTP.